backend/src/test/java/com/woowacourse/naepyeon/acceptance/MemberAcceptanceTest.java

```diff
@@ -1,39 +1,43 @@
 package com.woowacourse.naepyeon.acceptance;
 
 
-import static com.woowacourse.naepyeon.acceptance.AcceptanceFixture.회원_삭제;
-import static com.woowacourse.naepyeon.acceptance.AcceptanceFixture.회원_유저네임_수정;
-import static com.woowacourse.naepyeon.acceptance.AcceptanceFixture.회원_추가;
-import static com.woowacourse.naepyeon.acceptance.AcceptanceFixture.회원가입_후_로그인;
-import static org.assertj.core.api.Assertions.assertThat;
-
 import com.woowacourse.naepyeon.controller.dto.MemberRegisterRequest;
 import com.woowacourse.naepyeon.controller.dto.MemberUpdateRequest;
+import com.woowacourse.naepyeon.controller.dto.TokenRequest;
+import com.woowacourse.naepyeon.service.dto.MemberResponseDto;
 import com.woowacourse.naepyeon.service.dto.TokenResponseDto;
-import io.restassured.RestAssured;
 import io.restassured.response.ExtractableResponse;
 import io.restassured.response.Response;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
 import org.springframework.http.HttpStatus;
-import org.springframework.http.MediaType;
+
+import static com.woowacourse.naepyeon.acceptance.AcceptanceFixture.*;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
 
 class MemberAcceptanceTest extends AcceptanceTest {
 
     @Test
-    @DisplayName("회원 추가")
+    @DisplayName("회원을 추가하고 검증한다.")
     void addMember() {
-        //회원 생성
+        //회원 정보
         final MemberRegisterRequest member =
                 new MemberRegisterRequest("seungpang", "email@email.com", "12345678aA!");
-        final ExtractableResponse<Response> response = 회원_추가(member);
 
-        //회원이 추가됨
-        회원이_추가됨(response);
+        //회원 추가 및 토큰
+        ExtractableResponse<Response> response = 회원_추가(member);
```

Comment:
```suggestion
        final ExtractableResponse<Response> response = 회원_추가(member);
```
이 부분만 final이 없는 것 같네요!